.github/workflows/pr-title-checker.yaml

```diff
@@ -0,0 +1,25 @@
+name: "PR Title Checker"
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: thehanimo/pr-title-checker@v1.4.3
```

Comment:
Use hashes for github action versions https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash